lib/dependencies.js

```diff
@@ -21,6 +21,25 @@ const dependencies = [...internals, ...npmpkg, ...metapkg];
 
 const notLoaded = new Set();
 
+const isIdentifierRegex = /^[a-zA-Z_$][0-9a-zA-Z_$]*$/;
+
+const assignField = (container, name, value) => {
+  container[name] = value;
+  if (!isIdentifierRegex.test(name)) {
+    // If not a simple identifier try to adapt it to be accessible
+    // as an identifier by transforming - and / into upperCamelCase.
+    if (name.startsWith('@')) name = name.slice(1);
+    name = name
+      .split(/[/-]/)
+      .map((sub, i) => {
+        if (i === 0) return sub;
+        return sub.slice(0, 1).toUpperCase() + sub.slice(1);
+      })
+      .join('');
+    container[name] = value;
+  }
+};
```

Comment:
```suggestion
const spinalToCamel = (name) => {
  const words = name.split('-');
  const first = words.shift();
  return first + words.map(toUpperCamel).join('');
};

const assignNamespace = (container, name, value) => {
  container[name] = value;
  let key = name.startsWith('@') ? name.slice(1) : name;
  key = replace(key, '/', '-');
  key = spinalToCamel(key);
  container[key] = value;
};
```
p.s. We have `replace` and `toUpperCamel` in `metautils`

Reply:
And it would be good to put `spinalToCamel` to `metautils` as well

Reply:
`spinalToCamel` is now improved, landed but not published to npm

Reply:
Pushed suggested changes with a few improvements for now.